app/models/event.rb

```diff
@@ -154,6 +154,17 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # locks the ability to change application statuses
+  def lock_application_status
+    application_status_locked = true
```

Comment:
If you lock the status in db, why also lock it as a local variable if it's never used again?